### PR TITLE
added Cristina Badea tasks

### PR DESCRIPTION
**Pull request by BadeaCristina**

### CODE REVIEW

File: cristina-badea-dir/cristinabadea.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Cristina badea created this file during Lab 5
+second commit
+
```
Comment (Anisoara19) [inline cristina-badea-dir/cristinabadea.txt:6]: Add 'reviewed by Anisoara Poplacenel'

**Review by Anisoara19 — CHANGES_REQUESTED** [review]

File: cristina-badea-dir/cristinabadea.txt (hunk 1 above)
Comment (MelittaPirjolea) [inline cristina-badea-dir/cristinabadea.txt:6]: Add "reviewed by Melitta Pirjolea"

**Review by MelittaPirjolea — CHANGES_REQUESTED** [review]

**Review by BadeaCristina — COMMENTED** [review]
Review

**Review by BadeaCristina — COMMENTED** [review]
Added

**Review by Anisoara19 — APPROVED** [review]

**Review by MelittaPirjolea — APPROVED** [review]

**Pull request merged**